adapters/handlers/rest/handlers_objects.go

```diff
@@ -320,7 +325,7 @@ func (h *objectHandlers) updateObject(params objects.ObjectsClassPutParams,
 		principal, params.ClassName, params.ID, params.Body, repl, tenantKey)
```

Comment:
Same remark as for addObject, it would be more convenient to require the tenantKey to be part of the object itself rather than being part of query parameters. This would ensure that the tenantKey is always part of the object

Reply:
See [here](https://github.com/weaviate/weaviate/pull/3070#discussion_r1210299405)